database/helper.js

```diff
@@ -0,0 +1,13 @@
+const  model  = require('./index.js')
```

Comment:
Not a big deal atm, just a heads-up for when when you enable the AirBnB style guide. Since you're just grabbing the Restaurant model, you could use object destructuring here to get it directly. E.g. `const { Restaurant } = require('./index.js')`. That way you don't need to call `model.Restaurant` and can use Restaurant directly